index.textile

```diff
@@ -1,43 +1,27 @@
-h2. BlocklyProp help overview
+h3. Let's get started!
 
 p(short). BlocklyProp help overview
 
- <div class="col-sm-4 col-xs-12">
+<div class="col-sm-4 col-xs-12">
```

Comment:
Removing this space will result in showing the xml as text. Putting a space in front of a line ensures that the Textile converter does not change it into text. (While putting a space in front of a line using Textile markup will probably result in text that shows the markup text instead of applying it)